### PR TITLE
docs: edits to READMEs

### PR DESCRIPTION
**Pull request by jhaaaa**
- Elaborate on what and why for reactions
- Update remote attachment README to include install and preview attachment steps per Daria. Incorporating from this older PR: https://github.com/xmtp/xmtp-js-content-types/pull/7/files

### CODE REVIEW

File: packages/content-type-remote-attachment/README.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ Because XMTP messages can only be up to 1MB in size, we need to store the attach
 
 End-to-end encryption must apply not only to XMTP messages, but to message attachments as well. For this reason, we need to encrypt the attachment before we store it.
 
+### Install the package
+
+```bash
+npm i xmtp-content-type-remote-attachment
+```
+
```
Comment (rygine) [inline packages/content-type-remote-attachment/README.md:30]: a few things:

- the package was renamed to include the xmtp org in the scope. `xmtp-content-type-remote-attachment` => `@xmtp/content-type-remote-attachment`.

- can we add instructions for the 2 other package managers?

like this:

```bash
# npm
npm i @xmtp/content-type-remote-attachment

# yarn
yarn add @xmtp/content-type-remote-attachment

# pnpm
pnpm i @xmtp/content-type-remote-attachment
```

- can we move these instructions to the top after the descriptions?

- can we also add these installation instructions for the reactions content type?

File: packages/content-type-remote-attachment/README.md [hunk 2]
```diff
@@ -32,6 +38,18 @@ const attachment: Attachment = {
...
+Once you have the attachment object created, you can also create a preview for what to show in a message input before sending:
+
+```tsx
+URL.createObjectURL(
+              new Blob([Buffer.from(somePNGData)], {
+                type: attachment.mimeType,
+              }),
+            ),
+```
```
Comment (rygine) [inline packages/content-type-remote-attachment/README.md:51]: can we update the whitespace here for 2 space tabbed output?

File: packages/content-type-remote-attachment/README.md [hunk 3]
```diff
@@ -1,28 +1,38 @@
...
+
+```bash
+# npm
+npm i @xmtp/content-type-remote-attachment
+
+# yarn
+yarn add @xmtp/content-type-remote-attachment
+
+# pnpm
+
```
Comment (rygine) [inline packages/content-type-remote-attachment/README.md:31]: extra space here (and in the other one)

**Review by rygine — APPROVED** [review]

File: packages/content-type-remote-attachment/README.md (hunk 3 above)
Comment (jhaaaa) [reply packages/content-type-remote-attachment/README.md:31]: 🤦🏻‍♀️ thank you!

**Pull request merged**